frontend/src/app/password-change/password-change.component.spec.ts

```diff
@@ -117,7 +116,7 @@ describe('PasswordChangeComponent', () => {
     fixture.detectChanges();
 
     const userService = TestBed.get(CurrentUserService);
-    spyOn(userService, 'checkPassword').and.returnValue(Observable.throw(null));
+    spyOn(userService, 'checkPassword').and.returnValue(ErrorObservable.create(null));
```

Comment:
I get your point, but in the upcoming RxJS 6 it will be renamed `throwError` which is nicer, and I think importing `ErrorObservable` might be impossible.
Quoting from the [changelog](https://github.com/ReactiveX/rxjs/blob/master/CHANGELOG.md#breaking-changes):
```
* Observable: You should no longer deep import custom Observable implementations such as ArrayObservable or ForkJoinObservable.
* _throw: _throw is now exported as throwError
```

Reply:
I'll change it when RxJS6 comes out, then. We don't use it much, and it will be easy to change.